tests/PHPUnit/Unit/Scheduler/TimetableTest.php

```diff
@@ -61,6 +63,19 @@ public function testGetTimetableFromOptionValue($expectedTimetable, $option)
         self::resetPiwikOption();
     }
 
+    public function testRescheduleTaskAndRunTomorrow()
+    {
+        $timetable = new Timetable();
```

Comment:
@sgiehl I could review this if needed. What would you recommend me to do? Should I change this test for an Integration test or something similar?